cluster/addons/dns-horizontal-autoscaler/README.md

```diff
@@ -8,4 +8,4 @@ ConfigMap in `kube-system` namespace.
 
 Learn more about:
 - Usage: http://kubernetes.io/docs/tasks/administer-cluster/dns-horizontal-autoscaling/
```

Comment:
We could link to https://kubernetes.io/docs/concepts/cluster-administration/cluster-autoscaling/#sizing-a-workload-based-on-cluster-size too.

Reply:
Updated, thanks